doc/mix.exs

```diff
@@ -33,6 +33,7 @@ defmodule Doc.MixProject do
         "Architecture, Design and Concepts": ~r"/architecture/",
         "User Guide": ~r"/user/",
         "Administrator Guide": ~r"/administrator/",
+        "Operator Upgrade Guide": ~r"/upgrade/",
```

Comment:
Maybe just `Upgrade guide`? If I understand correctly @drf was suggesting that we put all the upgrade information here (like, e.g., API deprecations etc.)

Reply:
I just wanted to make it clear that those info refer to the operator (and not to Astarte upgrade). Does it make sense?

Reply:
I agree with @rbino TBH. It is also a bit misleading given you're upgrading also Astarte together with the Operator.